vault/resource_database_secret_backend_static_role_test.go

```diff
@@ -149,6 +150,55 @@ func TestAccDatabaseSecretBackendStaticRole_rotationSchedule(t *testing.T) {
 	})
 }
 
+func TestAccDatabaseSecretBackendStaticRole_Rootless(t *testing.T) {
+	backend := acctest.RandomWithPrefix("tf-test-db")
+	username := acctest.RandomWithPrefix("user")
+	dbName := acctest.RandomWithPrefix("db")
+	name := acctest.RandomWithPrefix("staticrole")
+	resourceName := "vault_database_secret_backend_static_role.test"
+
+	testRoleStaticCreate := `
+CREATE ROLE "{{name}}" WITH
+  LOGIN
+  PASSWORD '{{password}}';
+`
+
+	cleanup, pgxURL := testutil.PrepareTestContainerSelfManaged(t)
+	defer cleanup()
+
+	connURL := fmt.Sprintf("postgresql://{{username}}:{{password}}@%s/postgres?sslmode=disable", pgxURL.Host)
```

Comment:
nit: can we just use `pgxURL.String()`?

Reply:
I suppose in practice what we pass to the TF config will be the templated connection URL. So what you have here is technically correct. Going to resolve this one.